### PR TITLE
feat: add / update endpoints for post followups

### PR DESCRIPTION
**Pull request by Aylie-Chou**
# Issue
[[會員制]重要報導追蹤-後端實作](https://app.asana.com/0/1203699264568808/1207085276870885/f)

# Dependency
N/A

### CODE REVIEW

File: internal/mongo/document.go [hunk 1]
```diff
@@ -153,3 +161,64 @@ func BuildReviewLookupStatements() []bson.D {
...
 	return stages
 }
+
+func BuildFollowupLookupStatements(offset int, limit int) []bson.D {
+	var stages []bson.D
+
+	// match followups
+	stages = append(stages, bson.D{{
+		Key: StageMatch, Value: bson.D{
+			//{Key: "state", Value: "published"},
```
Comment (liruchen32) [inline internal/mongo/document.go:171]: Can we remove this line?

**Review by liruchen32 — CHANGES_REQUESTED** [review]
Just a small question

File: internal/mongo/document.go (hunk 1 above)
Comment (Aylie-Chou) [reply internal/mongo/document.go]: no, this line should be included 😂
updated in [e40f6bd](https://github.com/twreporter/go-api/pull/721/commits/e40f6bd3bc2094cc594186beb47536fb12f6aa06)

**Review by liruchen32 — APPROVED** [review]
LGTM

**Pull request merged**